src/rendervulkan.cpp

```diff
@@ -2352,9 +2352,12 @@ bool vulkan_composite( struct Composite_t *pComposite, struct VulkanPipeline_t *
 		.dstAccessMask = useForeignQueue ? (VkAccessFlagBits)0 : VK_ACCESS_MEMORY_READ_BIT,
 		.oldLayout = VK_IMAGE_LAYOUT_GENERAL,
 		.newLayout = VK_IMAGE_LAYOUT_GENERAL,
-		.srcQueueFamilyIndex = VK_QUEUE_FAMILY_IGNORED,
```

Comment:
Actually if we create with exclusive, we never should use `VK_QUEUE_FAMILY_IGNORED` so we can always set `srcQueueFamilyIndex` to `queueFamilyIndex` and the else part of `dstQueueFamilyIndex` to `queueFamilyIndex` too.

Sorry for being too lazy the first time ...

Reply:
Updated.